liubai-frontends/liubai-vscode-extension/src/utils/Logger.ts

```
@@ -0,0 +1,33 @@
+  static error(error: string | Error, ...args: any[]) {
+   this.outputChannel.error(error, ...args) 
+  }
+
+  static dispose() {
+    this.outputChannel.dispose()
+  }
+
+  static show() {
+    this.outputChannel.show()
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)